lib/linguist/heuristics.yml

```
@@ -478,6 +478,10 @@ disambiguations:
   - language: NL
     pattern: '^(b|g)[0-9]+ '
   - language: NewLisp
+- extensions: ['.nu']
+  rules:
+  - language: Nushell
+    pattern: '^\s*(import|export|module|def|let|let-env) '
```

We know the only other match is going to be Nu so we can shortcut things and save the classifier:

```suggestion
    pattern: '^\s*(import|export|module|def|let|let-env) '
  - language: Nu
```